java-tests/validating/src/test/kotlin/io/spine/test/options/setonce/SetOnceITest.kt

```diff
@@ -26,81 +26,118 @@
 
 package io.spine.test.options.setonce
 
-import io.spine.test.tools.validate.name
-import io.spine.test.tools.validate.setOnceExplicitFalse
-import io.spine.test.tools.validate.setOnceImplicitFalse
+import io.spine.test.options.setonce.TestEnv.CERF1
+import io.spine.test.options.setonce.TestEnv.CERF2
+import io.spine.test.options.setonce.TestEnv.DONALD
+import io.spine.test.options.setonce.TestEnv.EIGHTEEN
+import io.spine.test.options.setonce.TestEnv.EIGHTY
+import io.spine.test.options.setonce.TestEnv.EIGHTY_KG
+import io.spine.test.options.setonce.TestEnv.FIFTY_KG
+import io.spine.test.options.setonce.TestEnv.FIRST_YEAR
+import io.spine.test.options.setonce.TestEnv.JACK
+import io.spine.test.options.setonce.TestEnv.NO
+import io.spine.test.options.setonce.TestEnv.SHORT_HEIGHT
+import io.spine.test.options.setonce.TestEnv.SIXTEEN
+import io.spine.test.options.setonce.TestEnv.SIXTY
+import io.spine.test.options.setonce.TestEnv.STUDENT1
+import io.spine.test.options.setonce.TestEnv.STUDENT2
+import io.spine.test.options.setonce.TestEnv.TALL_HEIGHT
+import io.spine.test.options.setonce.TestEnv.THIRD_YEAR
+import io.spine.test.options.setonce.TestEnv.YES
+import io.spine.test.tools.validate.studentSetOnceFalse
+import io.spine.test.tools.validate.studentUnconstrained
 import io.spine.validation.assertions.assertValidationPasses
 import org.junit.jupiter.api.DisplayName
 import org.junit.jupiter.api.Test
 
+/**
+ * Tests for `(set_once)` constraint.
+ *
+ * Notice on a number of negative tests: they rely a bit on an implementation detail.
```

Comment:
With all the respect, I have no idea what you mean. Consider transforming this into a list with some simple points, or supplying with some links to the mentioned pieces of code ("higher level"?).